### DISCUSSION TITLE
Astro Static Extraction not working

### DISCUSSION THREAD

**Discussion by rundezv** [master-co/css#359] [Help] [answered]
https://github.com/master-co/css/tree/beta/examples/astro-with-static-extraction

I am trying this example, but I get the following error:

<img width="1265" alt="スクリーンショット 2024-06-13 8 34 37" src="https://github.com/master-co/css/assets/35029412/f341633b-4bd5-41fe-a876-8f4985dda775">

- astro v4.10.2
- @master/css v2.0.0-rc.40
- @master/css-extractor.vite v2.0.0-beta.160

**Comment by 1aron** [comment, marked as answer]
@takamoso This error is caused by the upstream vite deprecating this property. Fixed within two days.

<img width="1258" alt="Screenshot 2024-06-17 at 2 43 14 AM" src="https://github.com/master-co/css/assets/33840671/77bb7e8b-cf5e-451a-b6a7-deaf06a9ece1">

**Reply by 1aron** [reply]
Fixed in https://github.com/master-co/css/releases/tag/v2.0.0-rc.41
